overlord/ifacestate/helpers.go

```diff
@@ -429,6 +442,21 @@ func (m *InterfaceManager) autoConnect(task *state.Task, snapName string, blackl
 	return affectedSnapNames, nil
 }
 
+func getAutoconnectDisabled(st *state.State) (connFlags map[string]bool, err error) {
```

Comment:
can we instead store it in the state as a `[]string`, and load it into a map in this function?

Reply:
Good idea. Done.